src/bundles/ipfs-desktop.js

```diff
@@ -0,0 +1,8 @@
+const bundle = {
+  name: 'ipfs-desktop',
+  reducer: (state = {}) => state,
+  selectHasIpfsDesktop: () => !!window.ipfsDesktop,
+  ...(window.ipfsDesktop || {})
```

Comment:
It'd be a duplication, but I think we should spread the functions from ipfsDesktop that we expect to see here, so we can read the api of this bundle from the webui source code, rather than having to check elsewhere.